### PR TITLE
Doc/add documentation

### PR DESCRIPTION
**Pull request by Vertonowsky**
Added documentation for #81 and #82 issues.

### CODE REVIEW

File: src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/JoinRoom.java [hunk 1]
```diff
@@ -21,6 +21,84 @@
 import com.github.splendor_mobile_game.websocket.response.Result;
 
 // TODO: This whole class can be unit tested
+
+/**
+ * Reaction sent when new player joins to already created room.
```
Comment (kestivvi) [inline src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/JoinRoom.java]: Maybe I'm nitpicking, but it should be "...when the player **wants** to join an already...".

**Review by kestivvi — CHANGES_REQUESTED** [review]
Very good documentation. You have even said what is being validated. Please, resolve merge conflicts and we can merge it.

**Comment by Vertonowsky** [comment]
> Very good documentation. You have even said what is being validated. Please, resolve merge conflicts and we can merge it.

Thanks, I will do it later

**Review by kestivvi — APPROVED** [review]

**Review by agatamrnw — APPROVED** [review]
nice.

**Pull request merged**